### ISSUE TITLE
additionnal ca support for git fetch

### ISSUE THREAD

**Issue by poblin-orange** [rancher/fleet#150]
Using rancher 2.5.2 and the built in fleet continuous delivery, im not able to use or internal git repo (https, signed with corporate ca). I did configure rancher helm chart with additionnalCA, but it seems its not propagated to the fleet git fetch process

```
 https://gitea.xxx/paas-templates/gitops-fleet-repo.git refs/heads/master error: exit status 128, detail: fatal: unable to access 'https://gitea.xxxx/paas-templates/gitops-fleet-repo.git/': SSL certificate problem: unable to get local issuer certificate 
```
gz#13659

**Comment by izaac** [comment]
Reproduced on Rancher `master-head (12/02/2020)` _5ebc964_

I provisioned a Gitlab self hosted instance for testing. And I used certificates with private CA. Added also the certificate as described in the documentation:

https://rancher.com/docs/rancher/v2.x/en/installation/resources/chart-options/#additional-trusted-cas

**Comment by izaac** [comment]
Reopening until this is ready to test.

**Comment by StrongMonkey** [comment]
https://github.com/rancher/rancher/issues/29696#issuecomment-738959080 Due to this issue the rancher is not deploying the latest release of fleet. Should be fixed in https://github.com/rancher/rancher/pull/30404

**Comment by izaac** [comment]
Rancher version `v2.5-head (12/10/2020)` _baf4867_
`rancher/fleet:v0.3.2-rc3`


`SSL certificate problem: unable to get local issuer certificate`

**Comment by izaac** [comment]
in Fleet RC4 the git repos are stuck in `Git Updating`

**Comment by izaac** [comment]
Using a private CA is working now with `fleet:v0.3.2-rc5`

I've used the same Gitlab with private CA I used to reproduce this. But now GitRepo has support for adding a `caBundle`.

The Documentation was updated [here](http://fleet.rancher.io/gitrepo-add/). The CA should be added as a base64 string.

I was able to retrieve the bundles and get them deployed to a cluster.


Opened improvement for caBundle: https://github.com/rancher/fleet/issues/182
Opened dashboard support issue: https://github.com/rancher/dashboard/issues/2083
